components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -435,6 +437,8 @@ public static class RequestParams {
         public static final String CORRELATION_ID = "crId";
         public static final String IS_IDF_INITIATED_FROM_AUTHENTICATOR = "isIdfInitiatedFromAuthenticator";
         public static final String SESSION_ID = "sessionId";
+        public static final String CLIENT_ID_IN_REQUEST = "client_id";
```

Comment:
```suggestion
        public static final String CLIENT_ID = "client_id";
```

Since this is already in the class "RequestParams" we don't need to qualify this with  `_IN_REQUEST`